GitUI/NBugReports/BugReportInvoker.cs

```diff
@@ -104,11 +105,19 @@ public static void Report(Exception exception, bool isTerminating)
             }
 
             if (exception is ExternalOperationException externalOperationException
-                && externalOperationException.InnerException is { }
-                && externalOperationException.InnerException.Message.Contains(_dubiousOwnershipSecurityConfigString))
+                && externalOperationException.InnerException is Exception innerException)
             {
-                ReportDubiousOwnership(externalOperationException.InnerException);
-                return;
+                if (innerException.Message.Contains(_dubiousOwnershipSecurityConfigString))
+                {
+                    ReportDubiousOwnership(externalOperationException.InnerException);
+                    return;
+                }
+
+                if (innerException.Message.StartsWith(_fatalString))
+                {
+                    ReportGitFatal(externalOperationException, isTerminating);
+                    return;
+                }
             }
```

Comment:
What I have meant is reusing the existing handling, i.e. a few lines below just:
```
bool isUserExternalOperation = exception is UserExternalOperationException
    || exception is ExternalOperationException externalOperationException
        && externalOperationException?.InnerException?.Message?.StartsWith(_fatalString) is true;
```